src/stories/SubscriptAvatar.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import SubscriptAvatar from '../components/SubscriptAvatar';
+import * as Expensicons from '../components/Icon/Expensicons';
+import CONST from '../CONST';
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
+export default {
+    title: 'Components/SubscriptAvatar',
+    component: SubscriptAvatar,
+    args: {
+        mainAvatar: Expensicons.Profile,
+        secondaryAvatar: Expensicons.Workspace,
+        size: CONST.AVATAR_SIZE.DEFAULT,
+    },
+    argTypes: {
+        size: {
+            options: [CONST.AVATAR_SIZE.SMALL, CONST.AVATAR_SIZE.DEFAULT], // SubscriptAvatar only supports these two sizes
+        },
+    },
+};
+
+// eslint-disable-next-line react/jsx-props-no-spreading
+const Template = args => <SubscriptAvatar {...args} />;
+
+// Arguments can be passed to the component by binding
+// See: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Default = Template.bind({});
+
+const AvatarURLStory = Template.bind({});
+AvatarURLStory.args = {
+    mainAvatar: 'https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/avatar_1.png',
+    secondaryAvatar: 'https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/avatar_3.png',
```

Comment:
```suggestion
    mainAvatar: `${CONST.CLOUDFRONT_URL}/images/avatars/avatar_1.png`,
    secondaryAvatar: `${CONST.CLOUDFRONT_URL}/images/avatars/avatar_3.png`,
```
NAB: Is there any disadvantage to using the `CLOUDFRONT_URL` constant here?

Reply:
Oh good catch, thanks Amy! I will update the PR in a moment